pkg/foxtrot/auth_test.go

```diff
@@ -0,0 +1,39 @@
+package foxtrot
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRegisterLogin(t *testing.T) {
+	db, err := newDB("")
+	require.NoError(t, err)
+	defer db.close()
+
+	a := authenticator{db: db}
+	u := &User{Name: "Alice"}
+	err = a.register(context.Background(), u, "Pa$$w0rd")
```

Comment:
i would appreciate it if you would not add my password to the repository for all to see.

Reply:
👅